src/legacy/status_im/browser/core.cljs

```diff
@@ -443,9 +442,6 @@
         (= method "wallet_switchEthereumChain")
         (eip3326/handle-switch-ethereum-chain cofx dapp-name id message-id (first params))
 
-        (= method "wallet_addEthereumChain")
```

Comment:
removed this method - which I couldn't see how to access from "browser" but perhaps it's not?

Reply:
@pavloburykh, @qoqobolo - do you know if this is needed? 

Also @smohamedjavid if our current wallet can't handle it then does this really matter?
we have also removed wallet connect - not sure if that effects this here but probably this won't work anyway 🤔

Reply:
I guess it's safe to remove it as we don't allow users to add any Ethereum chain at this moment (at least foreseeable future)👍

Reply:
> @pavloburykh, @qoqobolo - do you know if this is needed?

@J-Son89 we do not know.

Reply:
> @pavloburykh, @qoqobolo - do you know if this is needed?

@J-Son89 we are not aware of the use of this method, sorry. 
I think it would be safer to check this with someone from the devs who implemented the old wallet

Reply:
Thanks, I think it's safe to remove for now